src/urh/signalprocessing/ProtocolAnalyzer.py

```diff
@@ -645,6 +646,14 @@ def from_xml_file(self, filename: str, read_bits=False):
         root = tree.getroot()
         self.from_xml_tag(root, read_bits=read_bits)
 
+    def to_pcapng(self, filename : str, hardware_desc_name: str = "", link_type: int = 147):
+        pcapng = PCAPNG()
```

Comment:
This is the only place where `PCAPNG` is used - no arguments are given to the constructor. Does it really need to be a class?